### PR TITLE
Now 2.0 deployment ready

### PR DESCRIPTION
**Pull request by paulogdm**
I just did a "copy & paste" of the previous `now.json` and also added a `.nowignore`.

### CODE REVIEW

File: now.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  ],
+  "routes": [
+    {
+      "src": "/main.js",
+      "dest": "/docs/main.js"
+    },
+    {
+      "src": "/(.*)",
+      "dest": "/docs/index.html"
+    }
```
Comment (jxnblk) [inline now.json]: Not that I plan to immediately, but this wouldn't pick up things like static assets, would it? I wonder if this last route can only do `.html` requests?

Comment (paulogdm) [reply now.json]: It seems you are correct.
I think we may have a bug on our "Output" of "/_src": 
image

Comment (paulogdm) [reply now.json]: I mean, does `mdx` uses any static assets after the build?

Comment (paulogdm) [reply now.json]: Yes, you are correct. But we can add routes later for that case. Something like:
```
    {
      "src": "/static/(.*)",
      "dest": "/docs/static/$1"
    }
```
But the fallback for `/docs/index.html` is still critical for a SPA.
https://github.com/zeit/now-examples/blob/master/create-react-app/now.json

**Comment by jxnblk** [comment]
Thanks for the help with this! I think for now I'm going to keep it on netlify – already have DNS/etc set up, but I'm sure I'll use this PR for future reference when I have to figure out how to get now set up

**Pull request closed without merging**